src/components/comment/Comment.jsx

```diff
@@ -37,7 +36,7 @@ const Comment = ({ comment }) => {
           </IconButton>
 
           <p className="post-numofvotes">
-            {comment.commentupvotes - comment.commentdownvotes}
+            {comment.downVotes - comment.upVotes}
```

Comment:
Should it be upVotes - downVotes instead

Reply:
fixed